### PR TITLE
Chore: Enhancing PR and Issue templates

### LINKED ISSUE #1762: Minor changes and enhancements in Issue and PR Template
There are some lines and para which should be commented in issue template but are not.
Also there should be a option to link `Github Issue` as well with `BugZilla Issue`

### PR DESCRIPTION
**Pull request by vishakhanihore**
Fixes #1762

### CODE REVIEW

**Comment by vishakhanihore** [comment]
@djmitche @helfi92 Please review.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,3 +5,7 @@
      and update this link.  Otherwise, just remove it from your PR comment.  -->
 
 Bugzilla Bug: [XXXXX](https://bugzilla.mozilla.org/show_bug.cgi?id=XXXXX)
+
+<!-- If this is related to a GitHub Bug/Issue, Please write Issue Number after # in the next line. Otherwise, just remove it from your PR comment. -->
+
+Github Bug/Issue: Fixes #XXXX
```
Comment (djmitche) [inline .github/pull_request_template.md]: :100:

Comment (helfi92) [reply .github/pull_request_template.md]: 👌

**Comment by helfi92** [comment]
@all-contributors please add @vishakhanihore for code, docs

**Pull request merged**